packages/protocol/contracts/adminnable/Adminnable.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import "./interfaces/IAdminnable.sol";
+
+/// @title Contract that implements multiple levels of admins
+contract Adminnable is IAdminnable {
```

Comment:
Should this be an abstract contracts with a non implemented getRank() function? I can see getRank being a useful method to have

Reply:
`adminToRank()` is exposed as `public` now so that's no longer needed. The previous implementation depended on not exposing that and overriding `getRank()` to implement difference in functionality between the contracts, that's no longer the case.